lib/hanami/model/migrator.rb

```diff
@@ -237,9 +237,9 @@ def self.version
       #
       # @since 0.7.0
       # @api private
-      def initialize(configuration: self.class.configuration)
+      def initialize(configuration: self.class.configuration, stream: $stdout)
```

Comment:
Wouldn't it be easier to pass the logger? I don't see that it's the Migrators responsibility to build it. Harder to configure logger/pattern like this, no?